src/box/box.h

```diff
@@ -626,7 +626,7 @@ API_EXPORT int
 box_session_push(const char *data, const char *data_end);
```

Comment:
Nit: `src: <...>` in the commit message doesn't look as a component. I would say `module api` or `doc`.

Reply:
Updated